pages/documentation/actions/change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: change
+---
+
+Change action mostly are used on textboxes and textareas, they are Triggering focus and change events on DOM element. Change action uses Jquery selector to Change the value of the DOM Element. Also users can use CBA placeholders:
+
+## Placeholders
+
+`_<$unique=>_` - Unique placeholder is generating random number (13 characters length maximum). So in the pic. example the action is used to find Dom Element with the ID of #sometextBox trigger focus event on it change the value to hello concatenated with 3 charecter length random number. 
+
+<img src='/images/Change.jpg' width = '600px'>
+
+`_<$clipboard=>_` - Clipboard placeholder is used to access [clipboard object](clipboard) data. So now you can access any data from clipboard object for change action using current syntax: <$clipboard=propertyName> so this placeholder will call clipboard\["propertyName"\]. 
+
+![](/images/clipboard-placeholder.jpg)
```

Comment:
I just resize images that are too big by its original size and extends or equals content  sizes. the other ones are just OK. This i suppose we might want to change if setting responsive properties.

Reply:
> I just resize images that are too big by its original size

They wouldn't stretch more than they parent content if you use for example `max-width: 100%` on images in content. So if you leave default behavior of width it will maximum shrink to 640px on desktop and adjust accordingly on mobile, so you don't need to adjust size for smaller screens. I think you want to stop hardcoding images. To test that you can change width of the main block to `640px`.